examples/mc_sc_workflow_example.md

```diff
@@ -63,6 +65,22 @@ Example output of newly created Vrf keypair:
 }
 ```
 
+**Model: Account**
+
+As well as the Vrf keypair and ed25519 keypair the EVM Account model requires a Secp256k1 keypair.
+
+For generating Secp256k1 key use command:  
+`generateAccountKey {"seed":"my seed"}`
+
+Example output of newly created Secp256k1 keypair:
+```
+{
+    "accountSecret":"05af34a8b28696763ca6870326969614194c853d69703a0cb7b97f6d92f4d97b2e",
+    "accountProposition":"e8674062598b86d8258257a4d79e99d15594d864"
+}
+```
+
+
 **Step 4: Generate data for ceased sidechain withdrawals proofs creation and verification**
```

Comment:
let's add the note that CSW is enabled only for the UTXO model and moreover only with the old certificate circuit (V1: without key rotation)